test/inductor/test_mps_basic.py

```diff
@@ -228,7 +228,44 @@ def forward(self, x):
         path = torch._inductor.aoti_compile_and_package(ep, "here.pt2")
         m = torch._inductor.aoti_load_package(path)
         res = m(*inp)
-        assert torch.allclose(res, res2)
+        self.assertTrue(torch.allclose(res, res2))
+
+    def test_const(self):
+        class M(torch.nn.Module):
+            def __init__(self) -> None:
+                super().__init__()
+                self.y = torch.randn(10, 10, device="mps")
+
+            def forward(self, x):
+                return x + self.y
+
+        inp = (torch.randn(10, 10, device="mps"),)
+        m = M().to("mps")
+        res2 = m(*inp)
+        ep = torch.export.export(m, inp)
+        path = torch._inductor.aoti_compile_and_package(ep, "here.pt2")
+        m = torch._inductor.aoti_load_package(path)
+        res = m(*inp)
+        self.assertTrue(torch.allclose(res, res2))
+
+    def test_two_const(self):
+        class Model(torch.nn.Module):
+            def __init__(self) -> None:
+                super().__init__()
+                self.y = torch.ones(3, 3, device="mps", dtype=torch.int32)
+                self.z = torch.ones(3, 3, device="mps", dtype=torch.int32) * 2
```

Comment:
Nit (also, why do you care about int32, strip dtype and let them be int64)
```suggestion
                self.z = torch.full((3, 3), 2, device="mps", dtype=torch.int32)
```